test/unit/Fixture/StaticPropertyGetSet.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+namespace BetterReflection\Fixture\StaticPropertyGetSet;
+
+class Foo
+{
+}
+
+class Bar
+{
+    public static $baz;
```

Comment:
Protected and private cannot be accessed (see https://3v4l.org/ftquS) so I have simply thrown an exception saying so for now